apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/preadj/R4ClaimResourceProviderIT.java

```diff
@@ -0,0 +1,142 @@
+
+    // Change random port value for consistent testing
+    actual = actual.replaceAll("localhost:\\d{4}", "localhost:6500");
+
+    assertEquals(expected, actual);
+  }
+
+  private static class TestResults {
+
+    public static final Date TEST_CREATED_DATE = Date.from(Instant.ofEpochMilli(1000));
+    public static final String TEST_BUNDLE_ID = "1856fb88-119e-46b6-a775-ab78ae0f61e9";
+
+    public static final String TEST_FISS =
+        "{\"resourceType\":\"Claim\",\"id\":\"f-123456\",\"meta\":{\"lastUpdated\":\"1970-01-01T00:00:00.000+00:00\"},\"contained\":[{\"resourceType\":\"Organization\",\"id\":\"provider-org\",\"meta\":{\"profile\":[\"http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-Organization\"]},\"identifier\":[{\"type\":{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/v2-0203\",\"code\":\"PRN\",\"display\":\"Provider number\"}]},\"value\":\"meda12345\"},{\"type\":{\"coding\":[{\"system\":\"http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBIdentifierType\",\"code\":\"npi\",\"display\":\"National Provider Identifier\"}]},\"system\":\"http://hl7.org/fhir/sid/us-npi\",\"value\":\"8876543211\"}]}],\"identifier\":[{\"type\":{\"coding\":[{\"system\":\"http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBIdentifierType\",\"code\":\"uc\",\"display\":\"Unique Claim ID\"}]},\"system\":\"https://dcgeo.cms.gov/resources/variables/dcn\",\"value\":\"123456\"}],\"status\":\"active\",\"type\":{\"coding\":[{\"system\":\"https://dcgeo.cms.gov/resources/codesystem/rda-type\",\"code\":\"FISS\"},{\"system\":\"http://terminology.hl7.org/CodeSystem/claim-type\",\"code\":\"institutional\",\"display\":\"Institutional\"}]},\"use\":\"claim\",\"patient\":{\"identifier\":{\"type\":{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/v2-0203\",\"code\":\"MC\",\"display\":\"Patient's Medicare number\"}]},\"system\":\"http://hl7.org/fhir/sid/us-mbi\",\"value\":\"123456MBI\"}},\"created\":\"1970-01-01T00:00:01+00:00\",\"provider\":{\"reference\":\"#provider-org\"},\"priority\":{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/processpriority\",\"code\":\"normal\",\"display\":\"Normal\"}]},\"diagnosis\":[{\"sequence\":1,\"diagnosisCodeableConcept\":{\"coding\":[{\"system\":\"http://hl7.org/fhir/sid/icd-10\",\"code\":\"princcd\"}]},\"type\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/ex-diagnosistype\",\"code\":\"principal\",\"display\":\"Principal Diagnosis\"}]}]},{\"sequence\":2,\"diagnosisCodeableConcept\":{\"coding\":[{\"system\":\"http://hl7.org/fhir/sid/icd-10\",\"code\":\"admitcd\"}]},\"type\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/ex-diagnosistype\",\"code\":\"admitting\",\"display\":\"Admitting Diagnosis\"}]}]}],\"procedure\":[{\"sequence\":1,\"date\":\"1970-07-20T00:00:00+00:00\",\"procedureCodeableConcept\":{\"coding\":[{\"system\":\"http://hl7.org/fhir/sid/icd-10\",\"code\":\"CODEABC\"}]}}],\"total\":{\"value\":1234.32,\"currency\":\"USD\"}}";
```

Comment:
would it be easier to manage these as separate JSON files?  In the coming months we'll be adding lots of new properties to these resources